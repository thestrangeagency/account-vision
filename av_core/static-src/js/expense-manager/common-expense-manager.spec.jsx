import React from 'react';
import { shallow } from 'enzyme';

import CommonExpenseManager from './common-expense-manager';

jest.mock('../services/expense-api-service');

test('it renders', () => {
  const wrapper = shallow(<CommonExpenseManager year="2017" />);
  expect(wrapper).toMatchSnapshot();
});
