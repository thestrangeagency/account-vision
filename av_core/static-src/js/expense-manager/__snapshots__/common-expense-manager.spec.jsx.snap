// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<form
  onSubmit={[Function]}
>
  <button
    className="btn btn-primary"
    disabled={true}
    type="submit"
  >
    Save
  </button>
</form>
`;
